manifest.appcache

```diff
@@ -1,5 +1,5 @@
-# Random number: 23
-# Random number (or any change) necessary to update client cache.
+# Versions (or any changes) are necessary to update client cache.
```

Comment:
This sounds as if the existence of a version was necessary to update the client cache.

Just put an instruction here to increment the version every time one wants to update the client cache.